NAMESPACE

```diff
@@ -80,9 +80,11 @@ export(uses_testthat)
 export(wd)
 export(with_debug)
 import(fs)
+import(rlang)
```

Comment:
This means we could remove lots of `rlang::`.

Reply:
I did this in most places, but there are quite a few instances of:

```r
check_dots_used(action = getOption("devtools.ellipsis_action", rlang::warn))
```

Where I kind of like the clarity using `rlang::`

Reply:
Yeah, I agree with you on that specific pattern.